tests/state_test_util.go

```diff
@@ -115,6 +119,12 @@ type stTransactionMarshaling struct {
 	PrivateKey hexutil.Bytes
 }
 
+var isTestExecutionSpecState bool
```

Comment:
Can you explain which problem was encountered(precompiled contract range mismatching?) or is expected if `isTestExecutionSpecState` disappears?

Reply:
Kaia limits address range of precompiled contracts by [this function](https://github.com/kaiachain/kaia/blob/08f4f7037822b1c255d40c90bdbde38da5382cf4/common/types.go#L354-L359). But some test cases set contracts whose address is like 0x00…00100 on setting pre state. So it is recognized as a reserved precompiled address on executing a transaction and then [this error](https://github.com/kaiachain/kaia/blob/08f4f7037822b1c255d40c90bdbde38da5382cf4/kerrors/kerrors.go#L36) is returned.